format/index.md

```diff
@@ -1756,10 +1760,14 @@ An error object **MAY** have the following members:
 [attributes]: #document-structure-resource-attributes
 [relationships]: #document-structure-resource-objects-relationships
 [resource relationships]: #document-structure-resource-objects-relationships
+[related resource link]: #document-structure-resource-objects-relationships
 [resource links]: #document-structure-resource-object-links
 [resource identifier object]: #document-structure-resource-identifier-objects
 [resource identifier objects]: #document-structure-resource-identifier-objects
 [fields]: #document-structure-resource-object-fields
 [error details]: #errors
 [member names]: #document-structure-member-names
+[link]: #document-structure-links
```

Comment:
no need to include more than one of these, you can do [anything at all][links] and it will work as long as there is an entry for [links]